### ISSUE TITLE
Auto font resize change box area size

### ISSUE THREAD

**Issue by hentaitaku** [xulihang/ImageTrans-docs#122]
Hi found a bug when auto font resize is enabled and in font settings vertical center is enabled will it change in Translated mode the box area height.
I use the font CCMightyMouth.

Ok have found out that it has todo with the image resize for the window when i use 100% is all ok but by 50% is it wrong.

**Comment by xulihang** [comment]
I found the position of the text box will change.

Vertical centering is achieved using the following code:

```vb
imageview.Top=parent,Height/2-imageview.PrefHeight/2
```

I seem to fix the problem with the following one:

```
imageview.Top=parent.PrefHeight/2-imageview.PrefHeight/2
```

**Comment by hentaitaku** [comment]
Nice say is there a way to give the background for the translation round corners when you not use text removal.

**Comment by xulihang** [comment]
I don't quite understand.

If not using precise text removal, the background will cover the source text.

![image](https://user-images.githubusercontent.com/5462205/147526670-f84b154c-0dbf-44d0-86e3-f2961f0a7c6f.png)

You can adjust the background by resizing the box. If in typesetting mode, the background will not be affected.

**Comment by hentaitaku** [comment]
I mean when i dont use precise text removal because it not removes text sometimes when i dont use precise text removal gets the text a background from imagetrans in the size from the select area like to make the borders there rounded.

**Comment by xulihang** [comment]
Okay. 

I made it draw rounded mask background. The radius has to be set manually. Please start an issue for this.

![image](https://user-images.githubusercontent.com/5462205/147528177-9306243f-54a7-484e-afe1-3b6e3d4872f8.png)

**Comment by xulihang** [comment]
This have been fixed in v1.5.3
